source/nodejs/adaptivecards/package-lock.json

```diff
@@ -3606,12 +3597,12 @@
 			}
 		},
 		"node_modules/marked": {
-			"version": "3.0.4",
-			"resolved": "https://registry.npmjs.org/marked/-/marked-3.0.4.tgz",
-			"integrity": "sha512-jBo8AOayNaEcvBhNobg6/BLhdsK3NvnKWJg33MAAPbvTWiG4QBn9gpW1+7RssrKu4K1dKlN+0goVQwV41xEfOA==",
+			"version": "4.0.12",
```

Comment:
Is it expected? I didn't see you update package.json

Reply:
this is expected -- `marked` is an indirect dependency of `typedoc`